_po/ja/how-to/laravel.po

```diff
@@ -815,6 +904,11 @@ msgid ""
 "search for free! Just type your keyword separate with space, you get AND searc"
 "h. If you separate words with OR (capital letter), you get OR search."
 msgstr ""
+"### 検索機能\n"
+"MySQLやPostgreSQLも前方一致ならインデックスを使ってLIKEで高速検索できますが、中間一致では"
+"インデックスを使ってくれません。PGroongaなら前方一致でも中間一致でも、インデックスを使って"
+"`LIKE`で高速検索できます！それだけではなく、特に苦労することなくAND検索やOR検索を使えます！"
+"スペースでキーワードを区切るとAND検索に、大文字のORで区切るとOR検索となります。"
```

Comment:
これもタイトルと説明の文を↓のように空行で区切っておくと

```markdown
### Search Function

Good thing about `PGroonga` is, unlike popular RDB like MySQL and PostgreSQL which only can use indexes for prefix match search, it can also use indexes for full-text partial match search! Not only that, you can get AND search and OR search for free! Just type your keyword separate with space, you get AND search. If you separate words with OR (capital letter), you get OR search.
```

翻訳対象が分割されます。

```po
msgid "### Search Function"

msgid ""
"Good thing about `PGroonga` is, unlike popular RDB like MySQL and PostgreSQL w"
"hich only can use indexes for prefix match search, it can also use indexes for"
" full-text partial match search! Not only that, you can get AND search and OR "
"search for free! Just type your keyword separate with space, you get AND searc"
"h. If you separate words with OR (capital letter), you get OR search."
```

今回はそういうメリットはないんですが、翻訳対象が分割されていると再利用できる可能性が高くなります。

この翻訳元と翻訳後のテキストのペアは全体で共有されているので、同じ字面の翻訳元は一回翻訳すればあとはそれを使い回せます。たとえば、`Since 1.0.0.`みたいなやつはいろんなところに出ているんですが、そういうやつは使いまわしています。

なので、基本的に、論理的に分割できるところ（タイトルと本文とか）は分割するのが好ましいです。

これも後で直しておくのでしばらくこのままで待ってください。
```
```suggestion
"### 検索機能\n"
"MySQLやPostgreSQLも前方一致ならインデックスを使ってLIKEで高速検索できますが、中間一致では"
"インデックスを使ってくれません。PGroongaなら前方一致でも中間一致でも、インデックスを使って"
"`LIKE`で高速検索できます！それだけではなく、特に苦労することなくAND検索やOR検索を使えます！"
"スペースでキーワードを区切るとAND検索に、大文字のORで区切るとOR検索となります。"
```

Reply:
TODO：後で`#+ ...`の行のあとに空行がなかったら報告するlintスクリプトを書いておくこと。